### ISSUE TITLE
Energy Legend: make all text black and add color swatches

### ISSUE THREAD

**Issue by arouinfar** [phetsims/masses-and-springs#359]
In https://github.com/phetsims/griddle/issues/49, `labelColor` was added to bar graphs, and the is default black. The primary motivation for this change was the contrast ratio between the colored text and the white background was poor (see values in https://github.com/phetsims/griddle/issues/49#issuecomment-622483092).

Another place that the colored text appears is in the Energy Legend:
![image](https://user-images.githubusercontent.com/8419308/80825430-9b4f1900-8b9d-11ea-8c24-1916fc8da6ef.png)

All text should be made black, but to compensate, we should add a color swatch next to each entry.
![image](https://user-images.githubusercontent.com/8419308/80825415-90948400-8b9d-11ea-9c1e-314bcce6a7de.png)

**Comment by Denz1994** [comment]
The color swatches have been added to the legend. Can @arouinfar review this change on master?

**Comment by arouinfar** [comment]
Looks great, thanks @Denz1994!

**Comment by Nancy-Salpepi** [comment]
In 1.1.0-dev.2, all text is now black and color swatches have been added. 

@arouinfar, I'm not sure the black color swatch for Total Energy makes sense anymore now that all of the labels are black. It's bar is a combo of all the other colors, not black. Thoughts?

EDIT: Actually, I see that the bar can be a dark gray when the spring is really stretched.
<img width="667" alt="Screenshot 2023-06-01 at 2 31 57 PM" src="https://github.com/phetsims/masses-and-springs/assets/87318828/942eccab-4408-4f0e-9bf8-95175a34820b">

**Comment by arouinfar** [comment]
Good find @Nancy-Salpepi. Black is definitely an inappropriate choice for the total Energy swatch. The stacked color representation only works if all of the values are positive, but the gravitational PE is sometimes negative. When that happens, we represent the total energy with a solid gray bar.

@jbphet let's change the total energy swatch color to the dark gray we use for the bar.

**Comment by arouinfar** [comment]
Tagging for https://github.com/phetsims/qa/issues/936

**Comment by jbphet** [comment]
Unassigning and marking as "blocks-sim-publication" since the republication of this sim has been deprioritized for now.  This should be addressed (at least to some degree) before republishing.
